### PR TITLE
Update hashes to use latest metadata

### PR DESCRIPTION
**Pull request by deitch**
Signed-off-by: Avi Deitcher <avi@deitcher.net>

**- What I did**
Update hashes in `examples/` and `docker-for-mac/` to use the latest `linuxkit/metadata` with `/run/config`

**- How I did it**
Changed hashes in 7 files

**- How to verify it**
Build and run images.

**- Description for the changelog**

Use `linuxkit/metadata` with `/run` support

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by deitch** [comment]
Needs a rebase. Will take care of it soon.

**Comment by deitch** [comment]
Phew. Rebase done. @ijc take another quick look?

File: docs/metadata.md [hunk 1]
```diff
@@ -104,6 +104,6 @@ and made available in `/run/config/userdata`.
 
 ## HyperKit
 
-HyperKit does not distiguish metadata and userdata, it's simply
+HyperKit does not distniguish metadata and userdata, it's simply
```
Comment (ijc) [inline docs/metadata.md]: This spelling is also wrong, "distinguish" is correct.

**Comment by ijc** [comment]
Your typo fix had a typo ;-)

File: docs/metadata.md (hunk 1 above)
Comment (deitch) [reply docs/metadata.md]: Ha! Fixed one typo with another!

**Comment by deitch** [comment]
> Your typo fix had a typo ;-)

Got it now!

**Comment by deitch** [comment]
> approved these changes

Thanks. Will merge after CI is done.

**Pull request merged**